ProfileManager.js

```diff
@@ -355,6 +370,29 @@ export default class ProfileManager {
       }));
     }
 
+    const profileAgentDetails = await this.getProfileAgent({profileAgent});
+    const documentReferenceId = edvs.getReferenceId('users-edv-document');
+    const capability = profileAgentDetails.zcaps[documentReferenceId];
```

Comment:
> In the system, there will be two capabilities for the profile agent with the *users-edv-document reference id upon profile creation. One for accessing the profile settings and one for accessing the profile agent's own user document. This code asserts that the capability we dereference is always the profile settings. Is this guaranteed?

Still have the same concern brought up in this comment https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/17#discussion_r397266190

Reply:
This should be addressed in another PR. There's currently no conflict because the zcaps with the same reference ID are not both stored in the same place (one is in the profileAgent record in the backend and one in the profileAgent's user doc in the EDV). We do need to fix this with something like `my-own-user-doc` vs. `the-profiles-user-doc` reference IDs, but we can do that later.